scripts/validate-build.sh

```
@@ -0,0 +1,483 @@
+    kubectl apply -k manifests/odh || \
+    log_warning "No kustomize manifests found"
+
+    log_info "Waiting for deployment..."
+    kubectl wait --for=condition=available --timeout=300s \
+        deployment/odh-dashboard -n opendatahub || true
+
+    kubectl get pods -A
+
+    POD=$(kubectl get pods -n opendatahub -l app=odh-dashboard -o name 2>/dev/null | head -1)
+    if [ -n "$POD" ]; then
+        kubectl logs -n opendatahub $POD --tail=50 || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)